deck-o-cards.js

```diff
@@ -1,33 +1,23 @@
-// 1. Make the function deck_o_cards assemble an array of cards using the provided suits and values arrays.
-// Each card in the deck should be an object formatted as: {suit: 'hearts', value: 'A'}
 function deck_o_cards() {
   var values = [2, 3, 4, 5, 6, 7, 8, 9, 10, 'Jack', 'Queen', 'King', 'Ace'];
   var suits = ['hearts', 'diamonds', 'clubs', 'spades'];
 
   var cards = []; // deck
   var shuffledCards = []; // deck shuffled
 
-  // Make 52 card objects and store them in the "cards" array
-  // Hint: use 2 for loops
+  for (var i = 0; i < suits.length; i++) {
 
+    for (var j = 0; j < values.length; j++) {
+        cards.push(values[j] + " " + 'of '+ suits[i]);
```

Comment:
You were definitely headed down the right path here but need to remember these are objects and not just strings in the array. Hopefully this morning's walkthrough helped clear some things up? If not - please let us know so we can answer your questions.